.travis.yml

```diff
@@ -95,13 +101,15 @@ before_deploy:
 - echo "deploying ${RELEASE_ZIP} to GitHub releases as tagged with ${TRAVIS_TAG}"
 - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sha256sum release/*.zip >> release/$RELEASE.txt; fi
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then shasum -a 256 release/*.zip >> release/$RELEASE.txt; fi
+- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then certutil -hashfile  release/*.zip SHA256 >> release/$RELEASE.txt; fi
 - export SHA_256_SUM="$(ls release/*.txt)";
 
 
 deploy:
   provider: releases
   skip_cleanup: true
-  api_key: $GITHUB_ACCESS_TOKEN
+   api_key:
+    secure: koxiDMX+Hl5fNd5TliLzxKI0H0eIs0elJVs686cfFNiyqnOsBS6wimxC2YK62wk/eXaHZ7gvi1Bfy6GuRnP2fNeLTL3IZ9O9HOcUP2X/jQ12fn5wkewr6DjQx3xlN+/BGSq9IAROMq5ol0moKFBFsu4rX28ovIl2Ft6TIJio8SflM7zZFI4WJa8NNGRBkkr0Dz+MB7fuMWrI3w0paj7BeHoz5ETsmW122plpAgaMQvmpB5k+ZhpCQYE0BwEEdXPAjYLOd6MQl99b0A+bg5ZC4/aJHC6UnCMdtgnzIGSrsNOdLCR8tz6tGv4rlf2TnxH9hKTKqz1N2GmQ4MSBukQpikb6DLcQbbOhjVSPwdWWU8y/O840Y68styYdTddrXJXRnr8sBtykxje1SeB3NaTsaIlSeJu8BfTgZXjJwI46kb3kSQsKHVTDdKfn3MeXEOQpxlZ4EHYEX/ZGc3Wk/uBeGpBMqiN0svdXxZTRaxl43jiJquGsI/3Kp5Q5XPlDp/cVPpCZJjQgw8qeS3fn14axnk373IJF9biSEVdfmfml5UjGX0MwMCip0OgFWvoGlo9VqMnnLYG2ltOI85XdMnZV2DuDPSUw7ODanYhnJFTr4035+TjxpXruO7wg6cuH6YPEUJR+I7pPeSUEX40RNvYsSKW07ZTLBLCkYArTcgRSiQ8=
```

Comment:
Does this then need to be updated before merge, to be a maidsafe repo specific key?